services/auth/basic.go

```diff
@@ -80,7 +80,7 @@ func (b *Basic) Verify(req *http.Request, w http.ResponseWriter, store DataStore
 			return nil
 		}
 
-		store.GetData()["IsApiToken"] = true
+		store.GetData()["IsApiToken"] = false
```

Comment:
Why this changed?

Reply:
I made `IsApiToken` to only indicate if it's authentication using a OAuth2 token. It's set to `true` in `services/auth/oauth2.go`, and `false` in `services/auth/basic.go` and `services/auth/httpsign.go`.

In `routers/api/v1/api.go` files, the `reqToken` function checks for three corresponding conditional cases.
1.  It checks if `IsApiToken` is `true`, which is when coming from `services/auth/oauth2.go`
2. It it's not, then it checks for `ctx.Context.IsBasicAuth`, corresponding to `services/auth/basic.go`
3. Finally, it checks for `ctx.IsSigned`, corresponding to `services/auth/httpsign.go`

Reply:
I think many places are using `IsApiToken`, it should be careful if we changed it's meaning.

Reply:
Ok, I somehow thought I introduced this, but it's not.. I'll revert them back